pkg/identity/identity.go

```diff
@@ -76,3 +79,87 @@ func (id *signingIdentity) Credentials() []byte {
 func (id *signingIdentity) Sign(message []byte) ([]byte, error) {
 	return id.sign(message)
 }
+
+func CreateSigner(privateKeyPath, certificatePath, mspID string) (SigningIdentity, error) {
+	priv, err := GetPrivateKey(privateKeyPath)
+	if err != nil {
+		return nil, err
+	}
+
+	cert, _, err := GetCertificate(certificatePath)
+	if err != nil {
+		return nil, err
+	}
+
+	return NewPrivateKeySigningIdentity(mspID, cert, priv)
+}
+
+func GetPrivateKey(f string) (*ecdsa.PrivateKey, error) {
+	in, err := os.ReadFile(f)
+	if err != nil {
+		return nil, err
+	}
+
+	k, err := PEMtoPrivateKey(in)
+	if err != nil {
+		return nil, err
+	}
+
+	key, ok := k.(*ecdsa.PrivateKey)
+	if !ok {
+		return nil, errors.New("expecting ecdsa key")
+	}
```

Comment:
The PEMtoPrivate key function used to get the key can return many key types. Why restrict it to only ECDSA keys here?